cirq/contrib/routing/__init__.py

```diff
@@ -18,10 +18,12 @@
     get_linear_device_graph,
     get_grid_device_graph,
     gridqubits_to_graph_device,
+    nx_qubit_layout,
```

Comment:
This name is not descriptive enough.

Reply:
I was attempting to make it look like networkx

```python
g = nx.Graph(...)
pos = nx.spring_layout(g)
pos = nx.planar_layout(g)
pos = nx_qubit_layout(g)
```